File: src/main/java/InputFileReader.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.util.List;

public class InputFileReader {
    public String readFromFile(String path) throws IOException {
        var wordsInFle = Files.readString(Path.of(path));
        return wordsInFle;
    }

    public List<String> getWordsExceptStopWords() throws IOException {
                var url = this.getClass().getClassLoader().getResource("stopwords.txt");
                assert url != null;
                return Files.readAllLines(Path.of(url.getPath()));
    }
}

File: src/main/kotlin/InputFileReader.kt
import java.nio.file.Files
import java.nio.file.Paths

class InputFileReader(private val fileName: String) {

    fun readText(): String {
        val path = Paths.get(fileName)
        if (!Files.exists(path)) {
            throw IllegalArgumentException("File not found: $fileName")
        }
        return Files.readString(path)
    }
    fun readTextFromResources(): String {
        val url = javaClass.classLoader.getResource(fileName)
            ?: throw IllegalArgumentException("Resource not found: $fileName")
        val path = Paths.get(url.toURI())

        if (!Files.exists(path)) {
            throw IllegalArgumentException("File not found: $fileName")
        }
        return String(Files.readAllBytes(path))
    }
}
